systest/Makefile

```diff
@@ -15,8 +15,6 @@
 
 # Inform make that targets are NOT files.
 .PHONY: all functest \
-install_tlc \
-install_test_infra \
```

Comment:
The logic that used to live in these targets/scripts is now encoded in Dockerfiles in the labbench repo.